File: java/darts/src/main/java/Darts.java
class Darts {

    int score(double xDart, double yDart) {
        double position = Math.abs(Math.sqrt(Math.pow(xDart, 2) + Math.pow(yDart, 2)));

        if (position <= 1) return 10;
        if (position <= 5) return 5;
        if (position <= 10) return 1;


        return 0;
    }
}

File: csharp/darts/Darts.cs
public static class Darts
{
    public static int Score(double x, double y)
    {
        var radius = Math.Sqrt(x * x + y * y);
        return radius switch
        {
            <= 1.0 => 10,
            <= 5.0 => 5,
            <= 10.0 => 1,
            _ => 0
        };
    }
}
